### PR TITLE
Windows/Linux build instruction language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Do you want me to add only the proper noun names in the Simplified Chinese versions?

### CODE REVIEW

File: docs/Compile/BuildWindows.md [hunk 1]
```diff
@@ -24,59 +25,59 @@ The previous version does not have enough support for cmake.
...
-    The previous version does not have enough support for cmake.
-  
-- git: [http://www.git-scm.com/](http://www.git-scm.com/)  
-  
-- cmake: [http://www.cmake.org/](http://www.cmake.org/)
+  + Qt (official release): https://download.qt.io/official_releases/qt/  
+    Current version: Qt 5.12.11
+  + IDE: Qt Creator. It is recommended to use version v4.15.0 or later. \
+  Prior versions don't have CMake support.
+
```
Comment (comradekingu) [inline docs/Compile/BuildWindows.md:39]:
```suggestion

```

File: docs/Compile/BuildWindows.md [hunk 2]
```diff
@@ -139,57 +140,57 @@ Specify CMake parameters: -Dtigervnc_DIR=[TigerVNC installation path]/lib/cmake
...
-  + FreeRDP_DIR: [freerdp installation path]/lib/cmake/FreeRDP2
-  + FreeRDP-Client_DIR: [freerdp installation path]/lib/cmake/FreeRDP-Client2
-  + tigervnc_DIR: [TigerVNC installation path]/lib/cmake
+  + RabbitCommon_DIR: RabbitCommon source-code location
+  + BUILD_FREERDP：If compiling FreeRDP
+  + WinPR_DIR: [FreeRDP installation path]/lib/cmake/WinPR2
+  + FreeRDP_DIR: [FreeRDP installation path]/lib/cmake/FreeRDP2
+  + FreeRDP-Client_DIR: [FreeFRP installation path]/lib/cmake/FreeRDP-Client2
+  + TigerVNC_DIR: [TigerVNC installation path]/lib/cmake
   + vncclient_DIR: [libvncserver installation path]/lib/cmake/LibVncServer
```
Comment (comradekingu) [inline docs/Compile/BuildWindows.md:173]: Not sure "lib/cmake/LibVncServer" is correct

File: docs/Compile/BuildLinux.md [hunk 3]
```diff
@@ -212,13 +212,13 @@ Specify CMake parameters: -Dtigervnc_DIR=[TigerVNC installation path]/lib/cmake
   + WinPR_DIR: [freerdp installation path]/lib/cmake/WinPR2
   + FreeRDP_DIR: [freerdp installation path]/lib/cmake/FreeRDP2
   + FreeRDP-Client_DIR: [freerdp installation path]/lib/cmake/FreeRDP-Client2
-  + tigervnc_DIR: [TigerVNC installation path]/lib/cmake
+  + TigerVNC_DIR: [TigerVNC installation path]/lib/cmake
   + vncclient_DIR: [libvncserver installation path]/lib/cmake/LibVncServer
```
Comment (comradekingu) [inline docs/Compile/BuildLinux.md:216]: Not sure "lib/cmake/LibVncServer" is correct

File: docs/Compile/BuildWindows.md [hunk 4]
```diff
@@ -91,8 +92,8 @@ If it is not in the same level directory, you must specify the CMake parameters:
 
 #### libvncserver
 - Compile from sorce code
-  + Source code location: [https://github.com/LibVNC/libvncserver](https://github.com/LibVNC/libvncserver)  
-  It is recommended to use patches: https://github.com/KangLin/libvncserver
+  + Source-code location: [https://github.com/LibVNC/libvncserver](https://github.com/LibVNC/libvncserver)  
+  It is recommended to use the patches from: https://github.com/KangLin/libvncserver
   + Specify CMake parameters: -Dvncclient_DIR=[libvncserver installation path]/lib/cmake/LibVncServer
```
Comment (comradekingu) [inline docs/Compile/BuildWindows.md:97]: Not sure "lib/cmake/LibVncServer" is correct

**Review by KangLin — APPROVED** [review]

File: docs/Compile/BuildWindows.md (hunk 2 above)
Comment (KangLin) [reply docs/Compile/BuildWindows.md:172]: I am will chanage to libVNCServer.

File: docs/Compile/BuildLinux.md (hunk 3 above)
Comment (KangLin) [reply docs/Compile/BuildLinux.md:216]: I am will chanage to libVNCServer.

File: docs/Compile/BuildWindows.md (hunk 4 above)
Comment (KangLin) [reply docs/Compile/BuildWindows.md:96]: I am will chanage to libVNCServer.

**Comment by KangLin** [comment]
> Do you want me to add only the proper noun names in the Simplified Chinese versions?

OK! Please help to modify:)

**Pull request merged**